### ISSUE TITLE
getting attachment error for rp_logger

### ISSUE THREAD

**Issue by Abhilash04** [reportportal/agent-python-pytest#150]
> test_main_page.py

```
@pytest.fixture(scope='function')
    def initialize(self, rp_logger):
        self.main_page = MainPage(self.driver)
        self.exe_status = ExecutionStatus(self.driver)
        self.ui_helpers = UIHelpers(self.driver)
        def cleanup():
            image = self.ui_helpers.take_screenshots(file_name_initials="report_portal")
            with open(image, "rb") as image_file:
                file_data = image_file.read()
            rp_logger.info("Screenshots", attachment={"name": "report_portal" + "." + str(round(time.time() * 1000)) + ".png",
                                           "data": file_data,
                                           "mime": "image/png"})
            self.driver.delete_all_cookies()
            rp_logger.info('Cleaning cookies.')
        yield
        cleanup()

```

> Errors

```
self = &lt;Logger conftest (DEBUG)&gt;, msg = 'Screenshots', args = ()
kwargs = {'attachment': {'data': b'\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x05\xa0\x00\x00\x03\t\x08\x06\x00\x00\x00\xf5c\x...5\xc6\xd8\x92\x05\x7f)\x00\x00\x00\x00IEND\xaeB`\x82', 'mime': 'image/png', 'name': 'report_portal.1574937279630.png'}}
    def info(self, msg, *args, **kwargs):
        """
        Log 'msg % args' with severity 'INFO'.
    
        To pass exception information, use the keyword argument exc_info with
        a true value, e.g.
    
        logger.info("Houston, we have a %s", "interesting problem", exc_info=1)
        """
        if self.isEnabledFor(INFO):
&gt; self._log(INFO, msg, args, **kwargs)
E TypeError: _log() got an unexpected keyword argument 'attachment'
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py:1383: TypeError
```

**Comment by iivanou** [comment]
@Abhilash04 could you please show the full stack trace?

**Comment by Abhilash04** [comment]
@iivanou 

https://gist.github.com/Abhilash04/3017833fb8b56007e9c48efed5adc1f9

**Comment by iivanou** [comment]
@Abhilash04 Your logger has not been patched. Can I see the full content of the test_main_page.py, please? Thanks.

**Comment by Abhilash04** [comment]
> Complete Project

https://github.com/Abhilash04/SeleniumPythonHybridFramework/tree/feature/reportportal

> test_main_page.py

```

""" This module contains the all test cases."""

import sys
import time
import allure
import pytest
from FrameworkUtilities.data_reader_utility import DataReader
from FrameworkUtilities.execution_status_utility import ExecutionStatus
from PageObjects.MainPageObjects.main_page import MainPage
from SupportLibraries.ui_helpers import UIHelpers

data_reader = DataReader()


@allure.story('[DEMO] - Automate  the  basic functionality')
@allure.feature('Web App Input Tests')
@pytest.mark.usefixtures("get_driver", "initialize")
class TestMainPage:
    """
    This class contains the executable test cases.
    """

    @pytest.fixture(scope='function')
    def initialize(self, rp_logger):
        self.main_page = MainPage(self.driver)
        self.exe_status = ExecutionStatus(self.driver)
        self.ui_helpers = UIHelpers(self.driver)

        def cleanup():
            image = self.ui_helpers.take_screenshots(file_name_initials="report_portal")
            with open(image, "rb") as image_file:
                file_data = image_file.read()
            rp_logger.info("Screenshots", attachment={"name": "report_portal" + "." + str(round(time.time() * 1000)) + ".png",
                                           "data": file_data,
                                           "mime": "image/png"})
            self.driver.delete_all_cookies()
            rp_logger.info('Cleaning cookies.')

        yield
        cleanup()

    @pytest.fixture(autouse=True)
    def class_level_setup(self, request):
        """
        This method is used for one time setup of test execution process.
        :return: it returns nothing
        """
        test_name = request.function.__name__
        if data_reader.get_data(test_name, "Runmode") != "Y":
            pytest.skip("Excluded from current execution run.")

    @pytest.mark.smoke
    @allure.testcase("Verify Main Screen Elements Test")
    def test_verify_main_screen_elements(self, rp_logger):
        """
        This test is validating the main screen elements. (positive scenario)
        :return: return test status
        """

        test_name = sys._getframe().f_code.co_name

        rp_logger.info("###### TEST EXECUTION STARTED :: " + test_name + " ######")

        with allure.step("Verify Main Screen Elements"):
            result = self.main_page.verify_main_screen_elements()
            self.exe_status.mark_final(test_step=test_name, result=result, logger=rp_logger)

    @pytest.mark.regression
    @allure.testcase("Valid User Input Test")
    def test_valid_user_input(self, rp_logger):
        """
        This test is validating the successful user input. (positive scenario)
        :return: return test status
        """

        test_name = sys._getframe().f_code.co_name

        rp_logger.info("###### TEST EXECUTION STARTED :: " + test_name + " ######")

        user_input = data_reader.get_data(test_name, "Text_Message")

        with allure.step("Verify User Input"):
            result = self.main_page.verify_valid_user_input(user_input)
            self.exe_status.mark_final(test_step=test_name, result=result, logger=rp_logger)

    @pytest.mark.smoke
    @allure.testcase("Valid Addition Test")
    def test_valid_addition(self, rp_logger):
        """
        This test is validating the successful addition. (positive scenario)
        :return: return test status
        """

        test_name = sys._getframe().f_code.co_name

        rp_logger.info("###### TEST EXECUTION STARTED :: " + test_name + " ######")

        num1 = data_reader.get_data(test_name, "Number_A")
        num2 = data_reader.get_data(test_name, "Number_B")
        expected_text = data_reader.get_data(test_name, "Expected")

        with allure.step("Verify valid addition functionality"):
            result = self.main_page.verify_addition_functionality(num1, num2, expected=expected_text)
            self.exe_status.mark_final(test_step=test_name, result=result, logger=rp_logger)

    @pytest.mark.regression
    @allure.testcase("Invalid Addition Test")
    def test_invalid_addition(self, rp_logger):
        """
        This test is validating the unsuccessful addition. (negative scenario)
        :return: return test status
        """

        test_name = sys._getframe().f_code.co_name

        rp_logger.info("###### TEST EXECUTION STARTED :: " + test_name + " ######")

        num1 = data_reader.get_data(test_name, "Number_A")
        num2 = data_reader.get_data(test_name, "Number_B")
        expected_text = data_reader.get_data(test_name, "Expected")

        with allure.step("Verify invalid addition functionality"):
            result = self.main_page.verify_addition_functionality(num1, num2, expected=expected_text)
            self.exe_status.mark_final(test_step=test_name, result=result, logger=rp_logger)

```

**Comment by iivanou** [comment]
@Abhilash04 I think the allure part of your tests makes manipulations with logging so that the default _log method is being restored to default. The code below works without issues on my side.

```
import pytest


@pytest.fixture(scope="session")
def rp_logger(request):
    import logging
    logger = logging.getLogger(__name__)
    logger.setLevel(logging.DEBUG)
    # Create handler for Report Portal if the service has been
    # configured and started.
    if hasattr(request.node.config, 'py_test_service'):
        # Import Report Portal logger and handler to the test module.
        from pytest_reportportal import RPLogger, RPLogHandler
        logging.setLoggerClass(RPLogger)
        rp_handler = RPLogHandler(request.node.config.py_test_service)
    else:
        import sys
        rp_handler = logging.StreamHandler(sys.stdout)
    # Set INFO level for Report Portal handler.
    rp_handler.setLevel(logging.INFO)
    return logger




@pytest.mark.usefixtures("initialize")
class TestMainPage:
    """
    This class contains the executable test cases.
    """

    @pytest.fixture(scope='function')
    def initialize(self, rp_logger):
        def cleanup():
            import subprocess
            free_memory = subprocess.check_output("free -h".split())
            rp_logger.info(
                "Case1. Memory consumption",
                attachment={
                    "name": "free_memory.txt",
                    "data": free_memory,
                    "mime": "application/octet-stream",
                },
            )
            rp_logger.info('Cleaning cookies.')

        yield
        cleanup()

    @pytest.mark.smoke
    def test_verify_main_screen_elements(self, rp_logger):
        """
        This test is validating the main screen elements. (positive scenario)
        :return: return test status
        """
        rp_logger.info("###### TEST EXECUTION STARTED :: " + 'test_verify_main_screen_elements' + " ######")
```

> pytest test.py --reportportal -s -vv
> platform linux -- Python 3.6.5, pytest-4.5.0, py-1.5.3, pluggy-0.11.0 -- /usr/bin/python3.6
> cachedir: .pytest_cache
> metadata: {'Python': '3.6.5', 'Platform': 'Linux-3.10.0-693.11.6.el7.x86_64-x86_64-with-centos-7.6.1810-Core', 'Packages': {'pytest': '4.5.0', 'py': '1.5.3', 'pluggy': '0.11.0'}, 'Plugins': {'xdist': '1.24.1', 'rerunfailures': '5.0', 'reportportal': '1.0.8', 'metadata': '1.7.0', 'lazy-fixture': '0.5.2', 'html': '1.19.0', 'forked': '0.2', 'cov': '2.5.1'}}
> rootdir: /root/gerrit/temp, inifile: pytest.ini
> plugins: xdist-1.24.1, rerunfailures-5.0, reportportal-1.0.8, metadata-1.7.0, lazy-fixture-0.5.2, html-1.19.0, forked-0.2, cov-2.5.1
> collected 1 item                                                                                                                                                                                                  
> test.py::TestMainPage::test_verify_main_screen_elements PASSED

![image](https://user-images.githubusercontent.com/8349078/69823327-9b777100-1219-11ea-9516-985edf1d4e8a.png)

**Comment by Abhilash04** [comment]
Hi @iivanou, I have created a simple project, where I want to attach some text files after each test case execution.

I am still getting the same error. 
`_log() got an unexpected keyword argument 'attachment'`

I have only 4 files under my projects. And I'm attaching requirements.txt for test purposes.
Please guide.

> 1. conftest.py

```
import pytest

@pytest.fixture(scope="session")
def rp_logger(request):
    import logging
    logger = logging.getLogger(__name__)
    logger.setLevel(logging.DEBUG)
    if hasattr(request.node.config, 'py_test_service'):
        from pytest_reportportal import RPLogger, RPLogHandler
        logging.setLoggerClass(RPLogger)
        rp_handler = RPLogHandler(request.node.config.py_test_service)
    else:
        import sys
        rp_handler = logging.StreamHandler(sys.stdout)
    rp_handler.setLevel(logging.INFO)
    return logger

```

> 2. pytest.ini

```
[pytest]

markers =
    smoke: quick tests that cover a good portion of the code
    unittest: unit tests for basic functionality
    integration: module to module functionality testing
    regression: critical area of the application which can break while adding new features


rp_uuid = XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
rp_endpoint = http://localhost:8080
rp_launch = API_Automation
rp_project = superadmin_personal
```

> 3. requirements.txt

```
# Framework
pytest

# Reporting
pytest-reportportal
```

> 4. test_api_responses.py

```

import sys
import pytest

@pytest.mark.usefixtures('initialize')
class TestAPI:

    @pytest.fixture(scope='function')
    def initialize(self, rp_logger):
        def cleanup():
            try:
                requirement_file = '/Users/Progressio/Documents/Personal/Github/DemoProject/requirements.txt'
                rp_logger.info(
                    "Case1. Memory consumption",
                    attachment={
                        "name": "requirements.txt",
                        "data": requirement_file,
                        "mime": "application/octet-stream",
                    },
                )
            except Exception as ex:
                rp_logger.error("Exception occured. \n{}".format(ex))
            
            rp_logger.info('Cleaning Test Data.')
        yield
        cleanup()

    @pytest.mark.smoke
    def test_verify_valid_response(self, rp_logger):

        test_name = sys._getframe().f_code.co_name
        rp_logger.info("###### TEST EXECUTION STARTED :: " +
                       test_name + " ######")

    @pytest.mark.regression
    def test_verify_invalid_response(self, rp_logger):

        test_name = sys._getframe().f_code.co_name
        rp_logger.info("###### TEST EXECUTION STARTED :: " +
                       test_name + " ######")

```

**Comment by Abhilash04** [comment]
@iivanou attaching the report portal screenshot and project itself.

![Screenshot 2019-11-29 at 3 00 50 PM](https://user-images.githubusercontent.com/17762842/69859214-bd283500-12b9-11ea-87d8-8685e929a8db.png)

[DemoProject.zip](https://github.com/reportportal/agent-python-pytest/files/3904576/DemoProject.zip)

**Comment by iivanou** [comment]
@Abhilash04 Can you remove your try/except block in the cleanup to see the full traceback? Thx.

**Comment by Abhilash04** [comment]
@iivanou here is the updated screenshot.

![image](https://user-images.githubusercontent.com/17762842/69867136-fa95be00-12cb-11ea-8239-136c46405a2f.png)

**Comment by Abhilash04** [comment]
@iivanou I have changed the MIME to **txt/plain** still no luck.

![image](https://user-images.githubusercontent.com/17762842/69867304-68da8080-12cc-11ea-9621-43e2f3ab5442.png)

**Comment by iivanou** [comment]
@Abhilash04  It is still not a traceback but a logged exception message. Below is an example of traceback I'm looking for:

> Traceback (most recent call last):
>   File "/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
>     self.run()
>   File "/usr/lib64/python3.6/threading.py", line 864, in run
>     self._target(*self._args, **self._kwargs)
>   File "/usr/lib/python3.6/site-packages/reportportal_client/service_async.py", line 89, in _monitor
>     self.handle(record)
>   File "/usr/lib/python3.6/site-packages/reportportal_client/service_async.py", line 70, in handle
>     handler(record)
>   File "/usr/lib/python3.6/site-packages/reportportal_client/service_async.py", line 223, in process_item
>     self.queue.qsize())
>   File "/usr/lib64/python3.6/logging/__init__.py", line 1294, in debug
>     self._log(DEBUG, msg, args, **kwargs)
>   File "/usr/lib/python3.6/site-packages/pytest_reportportal/rp_logging.py", line 120, in _log
>     return original_func(self, *args, **kwargs)
>   File "/usr/lib64/python3.6/logging/__init__.py", line 1425, in _log

Also, can you print out a list of loggers used in your code?

**Comment by Abhilash04** [comment]
@iivanou please find the details as requested.

> Loggers

```
<Logger dill (WARNING)>
<Logger urllib3.util.retry (NOTSET)>
<RPLogger urllib3.util (NOTSET)>
<Logger urllib3 (NOTSET)>
<Logger urllib3.connection (NOTSET)>
<Logger urllib3.response (NOTSET)>
<Logger urllib3.connectionpool (NOTSET)>
<Logger urllib3.poolmanager (NOTSET)>
<Logger requests (NOTSET)>
<Logger reportportal_client.service (NOTSET)>
<RPLogger reportportal_client (NOTSET)>
<Logger reportportal_client.service_async (NOTSET)>
<Logger pytest_reportportal.service (NOTSET)>
<RPLogger pytest_reportportal (NOTSET)>
<Logger pytest_reportportal.plugin (NOTSET)>
<Logger conftest (DEBUG)>
```
> traceback

```
2019-11-30 00:16:28
###### TEST EXECUTION STARTED :: test_verify_invalid_response ######
2019-11-30 00:16:28
http://localhost:8080 "POST /api/v1/superadmin_personal/item HTTP/1.1" 201 84
2019-11-30 00:16:28
http://localhost:8080 "POST /api/v1/superadmin_personal/item/5de16784a7b11b0001656531 HTTP/1.1" 201 84
2019-11-30 00:16:28
http://localhost:8080 "POST /api/v1/superadmin_personal/item/5de16784a7b11b0001656532 HTTP/1.1" 201 84
2019-11-30 00:16:28
http://localhost:8080 "POST /api/v1/superadmin_personal/log HTTP/1.1" 201 151
2019-11-30 00:16:29
def cleanup():
        requirement_file = '/Users/Progressio/Documents/Personal/Github/DemoProject/requirements.txt'
    
        try:
            rp_logger.info(
                "Requirement",
                attachment={
                    "name": "requirements.txt",
                    "data": requirement_file,
&gt; "mime": "txt/plain",
                },
            )

test_api_responses.py:25:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = &lt;Logger conftest (DEBUG)&gt;, msg = 'Requirement', args = ()
kwargs = {'attachment': {'data': '/Users/Progressio/Documents/Personal/Github/DemoProject/requirements.txt', 'mime': 'txt/plain', 'name': 'requirements.txt'}}

    def info(self, msg, *args, **kwargs):
        """
        Log 'msg % args' with severity 'INFO'.
    
        To pass exception information, use the keyword argument exc_info with
        a true value, e.g.
    
        logger.info("Houston, we have a %s", "interesting problem", exc_info=1)
        """
        if self.isEnabledFor(INFO):
&gt; self._log(INFO, msg, args, **kwargs)
E TypeError: _log() got an unexpected keyword argument 'attachment'

/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py:1383: TypeError

During handling of the above exception, another exception occurred:

self = &lt;test_api_responses.TestAPI object at 0x107e9b780&gt;, rp_logger = &lt;Logger conftest (DEBUG)&gt;

    @pytest.fixture(scope='function')
    def initialize(self, rp_logger):
        loggers = [logging.getLogger(name) for name in logging.root.manager.loggerDict]
        rp_logger.info(loggers)
        print(loggers)
    
        def cleanup():
            requirement_file = '/Users/Progressio/Documents/Personal/Github/DemoProject/requirements.txt'
    
            try:
                rp_logger.info(
                    "Requirement",
                    attachment={
                        "name": "requirements.txt",
                        "data": requirement_file,
                        "mime": "txt/plain",
                    },
                )
                rp_logger.info('Cleaning Test Data.')
    
            except Exception as ex:
                traceback.print_exc(ex)
    
        yield
&gt; cleanup()

test_api_responses.py:34:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
test_api_responses.py:31: in cleanup
    traceback.print_exc(ex)
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:163: in print_exc
    print_exception(*sys.exc_info(), limit=limit, file=file, chain=chain)
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:104: in print_exception
    type(value), value, tb, limit=limit).format(chain=chain):
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:508: in __init__
    capture_locals=capture_locals)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

klass = &lt;class 'traceback.StackSummary'&gt;, frame_gen = &lt;generator object walk_tb at 0x107e87660&gt;

    @classmethod
    def extract(klass, frame_gen, *, limit=None, lookup_lines=True,
            capture_locals=False):
        """Create a StackSummary from a traceback or stack object.
    
        :param frame_gen: A generator that yields (frame, lineno) tuples to
            include in the stack.
        :param limit: None to include all frames or the number of frames to
            include.
        :param lookup_lines: If True, lookup lines for each frame immediately,
            otherwise lookup is deferred until the frame is rendered.
        :param capture_locals: If True, the local variables from each frame will
            be captured as object representations into the FrameSummary.
        """
        if limit is None:
            limit = getattr(sys, 'tracebacklimit', None)
            if limit is not None and limit &lt; 0:
                limit = 0
        if limit is not None:
&gt; if limit &gt;= 0:
E TypeError: '&gt;=' not supported between instances of 'TypeError' and 'int'

/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:337: TypeError
```

**Comment by Abhilash04** [comment]
@iivanou additional traceback.

```
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/__init__.py:1383: TypeError

During handling of the above exception, another exception occurred:

self = &lt;test_api_responses.TestAPI object at 0x10a6999e8&gt;, rp_logger = &lt;Logger conftest (DEBUG)&gt;

    @pytest.fixture(scope='function')
    def initialize(self, rp_logger):
    
        def cleanup():
            requirement_file = '/Users/Progressio/Documents/Personal/Github/DemoProject/requirements.txt'
    
            try:
                rp_logger.info(
                    "Requirement",
                    attachment={
                        "name": "requirements.txt",
                        "data": requirement_file,
                        "mime": "txt/plain",
                    },
                )
                rp_logger.info('Cleaning Test Data.')
    
            except Exception as ex:
                traceback.print_stack(ex)
    
        yield
&gt; cleanup()

test_api_responses.py:30:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
test_api_responses.py:27: in cleanup
    traceback.print_stack(ex)
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:190: in print_stack
    print_list(extract_stack(f, limit=limit), file=file)
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:211: in extract_stack
    stack = StackSummary.extract(walk_stack(f), limit=limit)
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:344: in extract
    for f, lineno in frame_gen:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

f = TypeError("_log() got an unexpected keyword argument 'attachment'")

    def walk_stack(f):
        """Walk a stack yielding the frame and line number for each frame.
    
        This will follow f.f_back from the given frame. If no frame is given, the
        current stack is used. Usually used with StackSummary.extract.
        """
        if f is None:
            f = sys._getframe().f_back.f_back
        while f is not None:
&gt; yield f, f.f_lineno
E AttributeError: 'TypeError' object has no attribute 'f_lineno'

/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/traceback.py:298: AttributeError
```

**Comment by iivanou** [comment]
@Abhilash04 Your code works on my side:

> pytest test.py --reportportal -s -vv
> ============================== test session starts 
> platform linux -- Python 3.7.5, pytest-5.3.1, py-1.8.0, pluggy-0.13.1 -- /opt/venvs/3.7/bin/python3.7
> cachedir: .pytest_cache
> rootdir: /root/gerrit/temp, inifile: pytest.ini
> plugins: reportportal-1.0.8
> collected 2 items                                                                                                                                                                                                 
> 
> test.py::TestAPI::test_verify_valid_response [<Logger dill (WARNING)>,
>  <Logger urllib3.util.retry (NOTSET)>,
>  <RPLogger urllib3.util (NOTSET)>,
>  <Logger urllib3 (NOTSET)>,
>  <Logger urllib3.connection (NOTSET)>,
>  <Logger urllib3.response (NOTSET)>,
>  <Logger urllib3.connectionpool (NOTSET)>,
>  <Logger urllib3.poolmanager (NOTSET)>,
>  <Logger requests (NOTSET)>,
>  <Logger reportportal_client.service (NOTSET)>,
>  <RPLogger reportportal_client (NOTSET)>,
>  <Logger reportportal_client.service_async (NOTSET)>,
>  <Logger pytest_reportportal.service (NOTSET)>,
>  <RPLogger pytest_reportportal (NOTSET)>,
>  <Logger pytest_reportportal.plugin (NOTSET)>,
>  <Logger conftest (DEBUG)>]
> PASSED
> test.py::TestAPI::test_verify_invalid_response PASSED

![image](https://user-images.githubusercontent.com/8349078/70038947-249bf880-15ca-11ea-9e68-1362c5b21a27.png)

I'd suggest you try to run your code from the fresh virtualenv. Remove __pycache__ folder from the project as well.

**Comment by iivanou** [comment]
Fixed in #153 .
